### DISCUSSION TITLE
unsupported autoUnlockPlugWhenCharged

### DISCUSSION THREAD

**Discussion by costabel** [tillsteinbach/VWsFriend#118] [Q&A]
In the logs, I see recurringly:

vwsfriend_1   | 2021-12-09T15:59:57+0000:WARNING:charging_settings:An unsupported autoUnlockPlugWhenCharged: permanent was provided, please report this as a bug

..which I did herewith

**Comment by tillsteinbach** [comment]
Thank you for reporting, will be fixed in the next update!
